CMakeLists.txt

```diff
@@ -1,17 +1,23 @@
 cmake_minimum_required(VERSION 3.9)
 
-project(Ginkgo LANGUAGES C CXX VERSION 1.1.1 DESCRIPTION "A numerical linear algebra library targeting many-core architectures")
+project(Ginkgo LANGUAGES C CXX VERSION 1.2.0 DESCRIPTION "A numerical linear algebra library targeting many-core architectures")
```

Comment:
Will this automatically change `Ginkgo_VERSION_MAJOR` and so on?
Mainly asking to ensure that [`version_info::get_header_version()`](https://github.com/ginkgo-project/ginkgo/blob/develop/include/ginkgo/core/base/version.hpp#L216) returns the correct version.

Reply:
Yes that should work because we use `GKO_VERSION_MAJOR` etc in https://github.com/ginkgo-project/ginkgo/blob/daf1a4de99493914b16ea78f4fce014989d4e9fe/include/ginkgo/core/base/version.hpp#L216-L220 which is actually populated directly from CMake in the `config.hpp.in` file.